content/index.mdx

```diff
@@ -185,8 +185,7 @@ cml:
     python train.py </Tooltip>
 
     <Tooltip type="reports"> - cat metrics.txt >> report.md - cml publish
-    confusion_matrix.png --md >> report.md - cml send-comment report.md
-    </Tooltip>
+    plot.png --md >> report.md - cml send-comment report.md </Tooltip>
```

Comment:
a bit confused by the existing formatting here (and in the rest of this file) - @julieg18 are we missing some newlines?

Reply:
Yes, the formatting in this file is a little weird and hard to read, probably due to the mix of react components and markdown. Prettier helps in some sections but not everywhere. Sadly, I can't think of a way to improve it without manually formatting it. Is there an issue with the end result though? 

<img width="419" alt="image" src="https://user-images.githubusercontent.com/43496356/160123161-919836bc-51a6-40fb-92e0-576a8569b544.png">

Reply:
#213 thanks